moped-database/migrations/1703023666940_insert_into_public_moped_subphases/up.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO "public"."moped_subphases"("subphase_name", "subphase_description", "related_phase_id") VALUES (E'Discovery', null, 1);
+INSERT INTO "public"."moped_subphases"("subphase_name", "subphase_description", "related_phase_id") VALUES (E'Vetting', null, 4);
+INSERT INTO "public"."moped_subphases"("subphase_name", "subphase_description", "related_phase_id") VALUES (E'Commissioning', null, 7);
+INSERT INTO "public"."moped_subphases"("subphase_name", "subphase_description", "related_phase_id") VALUES (E'In-Service', null, 9);
+INSERT INTO "public"."moped_subphases"("subphase_name", "subphase_description", "related_phase_id") VALUES (E'Close-Out', null, 11);
```

Comment:
it is a bit misleading to have it here, but i'm cool leaving this since i assume it is an artifact of the auto-generated migrations that Hasura creates